src/main/java/org/folio/clients/InventoryClient.java

```diff
@@ -42,6 +43,11 @@
     return getReferenceDataByUrl(endpoint, params, IDENTIFIER_TYPES);
   }
 
+  public Map<String, JsonObject> getLocations(OkapiConnectionParams params) {
```

Comment:
Permissions needs to be added in module descriptor

Reply:
Updated